releasenotes/notes/fix-hybrid-visualization-layering-2c7e3d1a47d3d1d8.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed all circuit drawers to respect classical write ordering when scheduling hybrid operations,
+    so boxes that touch classical wires no longer overlap when registers are bundled, right justified,
+    or measurement arrows are disabled.

```

A few small suggestions for the release notes. The way they are written right now, they are a bit too technical. Remember they are intended for the users. Also the fixes (or at least the tests) were only for the text circuit drawer.
Something like this:

```suggestion
    Fixed the circuit text drawer so that circuit instructions with classical wires are drawn in separate layers.
```